static/bitbucket-brands.svg

```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" data-prefix="fab" data-icon="bitbucket" class="svg-inline--fa fa-bitbucket fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><symbol id="bitbucket" viewBox="0 0 512 512"><path fill="currentColor" d="M23.1 32C14.2 31.9 7 38.9 6.9 47.8c0 .9.1 1.8.2 2.8L74.9 462c1.7 10.4 10.7 18 21.2 18.1h325.1c7.9.1 14.7-5.6 16-13.4l67.8-416c1.4-8.7-4.5-16.9-13.2-18.3-.9-.1-1.8-.2-2.8-.2L23.1 32zm285.3 297.3H204.6l-28.1-146.8h157l-25.1 146.8z"></path></symbol></svg>
```

Comment:
I think it's better to use CSS as suggested in https://fontawesome.com/how-to-use/on-the-web/setup/getting-started?using=web-fonts-with-css
Include their CSS via their CDN, and show the icons using `<i>` elements.